doc/content/en/docs/++version++/Specification/_index.md

```diff
@@ -825,18 +825,28 @@ _Currently only available in Java and Rust_.
 Here, as scale property is stored in value itself it needs more bytes than preceding `decimal` type, but it allows more flexibility.
 
 ### UUID
+
 The `uuid` logical type represents a random generated universally unique identifier (UUID).
 
-A `uuid` logical type annotates an Avro `string`. The string has to conform with [RFC-4122](https://www.ietf.org/rfc/rfc4122.txt)
+A `uuid` logical type annotates an Avro `string` or `fixed` of length 16. The string has to conform with [RFC-4122](https://www.ietf.org/rfc/rfc4122.txt)
```

Comment:
As RFC 4122 specifies both the string syntax as the byte layout, I propose one of this change:
```suggestion
A `uuid` logical type annotates an Avro `string` or `fixed` of length 16. Both the string and `fixed` byte layout have to conform with [RFC-4122](https://www.ietf.org/rfc/rfc4122.txt).
```

A question though: as we're not specifying anything here, is it clear that there is no "explicit application or presentation protocol specification to the contrary" (as mentioned in the RFC, in [section 4.1.2](https://www.ietf.org/rfc/rfc4122.html#section-4.1.2)?

Reply:
I like the proposal, thanks for that. Regarding the question; it is clear to me :)